### PR TITLE
IntaRNA : enable multithreading (linux)

### PR DESCRIPTION
**Pull request by martin-mann**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/intarna/meta.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ requirements:
   run:
     - libgcc # [linux]
     - libcxx # [osx]
-    - boost {{CONDA_BOOST}}* # [linux]
+    - boost =={{CONDA_BOOST}}* # [linux]
```
Comment (bgruening) [inline recipes/intarna/meta.yaml]: the `==` is not needed.

**Comment by martin-mann** [comment]
Moin @bgruening , any idea why linting says "boost not pinned"? Worked so far, what has changed?

**Comment by martin-mann** [comment]
might the problem occure due to the non-CONDA_BOOST version I am pinning for OSX?

**Comment by martin-mann** [comment]
nope, even then the same error...

**Comment by martin-mann** [comment]
also not working

**Comment by martin-mann** [comment]
just found that CONDA_BOOST is now 1.60 too.. so no distinction should be needed.. right

but anyway.. seems to be a problem with the linting if system specific pinning is not working... or?

**Pull request merged**